src/static/newtab.js

```diff
@@ -0,0 +1,8 @@
+browser.storage.sync.get("userconfig").then(c => {
+    if (c.userconfig.newtab) {
+        document.location = c.userconfig.newtab
```

Comment:
You need to force this to a URI, as `excmds.open(newtab)` did, otherwise `set newtab news.bbc.co.uk` and friends won't work.

Reply:
Thanks for catching this.